recipe/meta.yaml

```diff
@@ -15,7 +15,7 @@ source:
   {{ hash_type }}: {{ hash }}
 
 build:
-  number: {{ build }}
+  number: 0
```

Comment:
Any thoughts on this @CJ-Wright? Can see how people value having everything (including build number at the top). Does this seem reasonable to you to support?

Reply:
Yea the bot just needs a smarter regex and a new one to match the build number variable.